packages/features/eventtypes/components/CreateEventTypeDialog.tsx

```diff
@@ -254,6 +254,7 @@ export default function CreateEventTypeDialog({
                   placeholder={t("quick_video_meeting")}
                   firstRender={firstRender}
                   setFirstRender={setFirstRender}
+                  maxHeight="250px"
```

Comment:
This isn't needed, dialog shouldn't break even if you extend the height of textarea beyond 250px. We should focus on fixing that issue and not limiting here.

Reply:
no this is for making the initial height to be fixed only i mean when we initially write something then don't grow it very long and we can overwrite it by the resize icon at bottom left.

Reply:
and i use this css <DialogFooter showDivider className="-ml-1 -mr-1 border-l-4 border-r-4"> to hide/fix the issue